flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentInternal.java

```diff
@@ -74,15 +75,15 @@ public interface TableEnvironmentInternal extends TableEnvironment {
      * @param operations The operations to be executed.
      * @return the affected row counts (-1 means unknown).
      */
-    TableResultInternal executeInternal(List<ModifyOperation> operations);
+    TableResult executeInternal(List<ModifyOperation> operations);
 
     /**
      * Execute the given operation and return the execution result.
      *
      * @param operation The operation to be executed.
      * @return the content of the execution result.
      */
-    TableResultInternal executeInternal(Operation operation);
+    TableResult executeInternal(Operation operation);
```

Comment:
Why do we need to change this?

Reply:
Previous, we have `ExtendedOperationExecutor#executeOperation` marked as `PublicEvolving`, so the result of this method should be TableResult which is also marked as `PublicEvolving`. So we should also change it to `TableResult` in here.

Now, I revert this changes since we remove `ExtendedOperationExecutor`.